cmd/boostd/init.go

```diff
@@ -467,6 +487,7 @@ func migrateMarketsConfig(cctx *cli.Context, mktsRepo lotus_repo.LockedRepo, boo
 		rcfg.LotusDealmaking = mktsCfg.Dealmaking
 		rcfg.LotusFees = mktsCfg.Fees
 		rcfg.DAGStore = mktsCfg.DAGStore
+		rcfg.DAGStore.RootDir = ""
```

Comment:
```suggestion
		// Clear the DAG store root dir config, because the DAG store is no longer configurable in Boost
		// (it is always at <repo path>/dagstore
		rcfg.DAGStore.RootDir = ""
```